e2e/dockerutil/dockerutil.go

```diff
@@ -19,7 +18,7 @@ const testLabel = "ibc-test"
 // note: the test suite name must be passed as the chains are created in SetupSuite which will label
 // them with the name of the test suite rather than the test.
 func GetTestContainers(ctx context.Context, suiteName string, dc *dockerclient.Client) ([]dockertypes.Container, error) {
-	testContainers, err := dc.ContainerList(ctx, container.ListOptions{
+	testContainers, err := dc.ContainerList(ctx, dockertypes.ContainerListOptions{
```

Comment:
does this need to be reverted?

Reply:
For now, yes. We need to stay on the same docker go library as interchaintest.
I would want to update interchaintest a bit, so might fix that then.